subcell_pipeline/analysis/compression_metrics/compression_analysis.py

```diff
@@ -208,21 +255,27 @@ def get_contour_length_from_trace(
 
 def get_bending_energy_from_trace(
     polymer_trace: np.ndarray,
-    **options: dict,
-) -> float:
+    **options: dict[str, Any],
+) -> Union[float, np.floating[Any]]:
     """
-    Returns the bending energy per monomer of a polymer trace.
+    Calculate the bending energy per monomer of a polymer trace.
 
     Parameters
     ----------
-    polymer_trace: [n x 3] numpy array
+    polymer_trace
         array containing the x,y,z positions of the polymer trace
-    **options: dict
+
+    **options
         Additional options as key-value pairs.
```

Comment:
This docstring is not getting parsed correctly. I think you need a newline between `Additional options as key-value pairs.` and `bending_constant: float`. Alternatively, I think you can just promote the kwargs to actual arguments, so they are directly used:

```python
def get_bending_energy_from_trace(
    polymer_trace: np.ndarray,
    bending_constant: float = 1.0,
) -> Union[float, np.floating[Any]]:
```

It should still work with the generic `functions[self](polymer_trace, **options)` as long as the arguments have a default value.

Reply:
Fixed in [f8ffc9](https://github.com/simularium/subcell-pipeline/pull/58/commits/f8ffc9bc7f8a2e06701ffbdfdb6849aa836ee4b2)

I opted to keep the kwargs to have a uniform way of passing arguments to the compression metric calculations. Promoting to actual arguments may cause errors if unexpected keywords are received as input.